chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/SummaryReporter.java

```diff
@@ -168,7 +168,6 @@ private void getData(String sessionId) {
 
             Set<String> standardMetricsIds = new HashSet<String>();
             standardMetricsIds.addAll(StandardMetricsNamesUtil.getAllVariantsOfMetricName(StandardMetricsNamesUtil.THROUGHPUT_ID));
-            standardMetricsIds.addAll(StandardMetricsNamesUtil.getAllVariantsOfMetricName(StandardMetricsNamesUtil.FAIL_COUNT_ID));
             standardMetricsIds.addAll(StandardMetricsNamesUtil.getAllVariantsOfMetricName(StandardMetricsNamesUtil.SUCCESS_RATE_ID));
             standardMetricsIds.addAll(StandardMetricsNamesUtil.getAllVariantsOfMetricName(StandardMetricsNamesUtil.LATENCY_ID));
```

Comment:
What about backward compatibility? If though I can delete all synonyms for all metrics.